### PR TITLE
Fixes #117. Update header.html

### LINKED ISSUE #117: Add google code-in in the fossasia events dropdown
Add linked google code in sites of last two years and the link to actual gci page of fossasia

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Fixes #117 
Updated header.html
Added google code  and google summer of code tabs to event dropdwon
screen shot 2017-12-30 at 12 23 59 pm

### CODE REVIEW

**Comment by saarthakchats** [comment]
Deploy link?

**Comment by arjunpandey2003** [comment]
https://arjunpandey2003.github.io/2012.fossasiaunicodef.org/.

**Comment by vaibhavsingh97** [comment]
@arjunpandey2003 Please squash your commit and use some meaningful commit message. Please change your PR title to Fixes #<issue-number: <commit-message>. Please follow best practices

File: _includes/header.html [hunk 1]
```diff
@@ -28,6 +28,9 @@
                 <li><a target="_self" href="http://2014.fossasia.org">FOSSASIA Summit 2014</a></li>
                 <li><a target="_self" href="http://2011.fossasia.org">FOSSASIA Summit 2011</a></li>
                 <li><a target="_self" href="http://2010.fossasia.org">FOSSASIA Summit 2010</a></li>
+                <li><a target="_self" href="http://gci17.fossasia.org">GCI Official Site</a></li>
```
Comment (vaibhavsingh97) [inline _includes/header.html:31]: These are not FOSSASIA Events

File: _includes/header.html [hunk 2]
```diff
@@ -28,6 +28,9 @@
...
+                <li><a target="_self" href="https://summerofcode.withgoogle.com/archive/2017/organizations/5936828200779776/">GSoC Fossasia</a></li>
```
Comment (vaibhavsingh97) [inline _includes/header.html:32]: same

File: _includes/header.html (hunk 1 above)
Comment (vaibhavsingh97) [reply _includes/header.html:31]: This can come under `Quick links`

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**